angular/angular/README.md

```diff
@@ -4,24 +4,17 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
 
 ## Development server
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+Run `docker compose up -d` for a dev server.
+Navigate to `http://localhost:4200/`.
+The app will automatically reload if you change any of the source files.
 
-## Code scaffolding
+## Use with Docker Development Environments
 
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
+You can use this sample with the Dev Environments feature of Docker Desktop.
 
-## Build
+![Screenshot of creating a Dev Environment in Docker Desktop](../dev-envs.png)
```

Comment:
@glours I put a generic screenshot (no URL pre-filled) in the repo root. I don't know that it's worth having example-specific screenshots both because it's tedious 😛 but also at ~200KB each, that'll be several extra MB in Git blobs across all examples

Reply:
I think this section will be a temporary one, so maybe you're right adding a generic capture for all the samples should be better